assets/css/_vehicle_properties_card.scss

```diff
@@ -0,0 +1,121 @@
+// #region Vehicle Properties Card
+.m-vehicle-properties-card {
+  box-sizing: border-box;
+
+  *:before,
+  *:after,
+  * {
+    box-sizing: inherit;
+  }
+}
+
+.m-vehicle-properties-card {
+  font-family: $font-family;
+
+  // Static Design Width per spec
+  width: 395px;
+  // Card Look
+  border-radius: 8px;
+  background-color: $color-gray-100;
+  border: 1px solid $color-gray-400;
+  // Clip children to keep rounded edges for background color
+  overflow: clip;
+
+  filter: drop-shadow(0px 8px 16px rgba(0, 0, 0, 0.16))
+    drop-shadow(0px 16px 24px rgba(176, 181, 192, 0.24));
+
+  // all children expand to full width of card
+  > * {
+    width: 100%;
+  }
+}
+
+.m-vehicle-properties-card__title-bar {
+  background-color: $white;
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.m-vehicle-properties-card__body {
+  display: flex;
+  flex-flow: column;
+  gap: 10px;
+}
```

Comment:
```suggestion
.m-vehicle-properties-card__body > :not(:first_child) {
	margin-top: 10px
}
```

Could be simplified

Reply:
Really struggling between what design has specified in figma vs what's "better"?

I _feel_ like the box model is faster than flex in this context, but this is an assumption.

Design used flex when building in figma, so it feels natural to reach for flex as if it's a specification from design, but design also says they won't recommend technical solutions, so this is up to us in the end?

Reply:
I'd reach out to design, but I think in general design's choices about the method of layout in Figma aren't super important, it's more a question of getting the ultimate visual output.